elastic/kibana
# [Ingest Pipelines] Pipelines details header improvements

Closes https://github.com/elastic/kibana/issues/179890

## Summary

This PR improves the UX of the page header of the pipeline details page by bundling certain actions that can be performned on the pipeline in a single place while increasing readability for the user.

##### How to test

* Start up kibana and es
* Navigate to Stack management -> Ingest pipelines
* Create a new pipeline
* Verify that:


<details>
<summary>Screenshots</summary>

// Create pipeline
![Screenshot 2024-04-03 at 14 52 47](https://github.com/elastic/kibana/assets/1191206/c86d517d-72ab-4efe-8d7d-40dedf6bb71e)

// Edit pipeline
![Screenshot 2024-04-03 at 14 53 12](https://github.com/elastic/kibana/assets/1191206/d6aee357-a157-4f02-bb76-5ed4ca8ea97d)

</details>
